src/styles/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$color-day: #eee;
+$color-hover: #88c928;
```

Comment:
![image](https://user-images.githubusercontent.com/127235037/234551159-e2218ff8-46e2-4029-93ff-7c869a31717b.png)

$color-hover has to be pink according to the task.